src/Integration.Vsix/Protobuf/build.bat

```diff
@@ -1,2 +1,4 @@
-set toolspath=..\..\..\packages\Grpc.Tools.1.2.2\tools\windows_x64
+set toolspath=..\..\..\packages\Grpc.Tools.1.4.1\tools\windows_x64
 %toolspath%\protoc.exe -I=. --csharp_out . --grpc_out . sonarlint-daemon.proto --plugin=protoc-gen-grpc=%toolspath%\grpc_csharp_plugin.exe
+
+pause
```

Comment:
I don't think it is called by the build. Generated files are commited in Git, so generating them is a manual process. I added pause to more easily see the output.